test/jdk/java/awt/Multiscreen/MultiScreenLocationTest/MultiScreenLocationTest.java

```diff
@@ -103,6 +103,16 @@ public static void main(String[] args) throws AWTException
                 || image.getRGB(image.getWidth() - 1, 0) != rgb
                 || image.getRGB(image.getWidth() - 1, image.getHeight() - 1) != rgb
                 || image.getRGB(0, image.getHeight() - 1) != rgb) {
+                    System.out.println("Robot.createScreenCapture Expected: " +
+                            String.format("0x%08X",rgb));
+                    System.out.println("Robot.createScreenCapture Top Left Actual: " +
+                            String.format("0x%08X",image.getRGB(0, 0)));
+                    System.out.println("Robot.createScreenCapture Top Right Actual: " +
+                            String.format("0x%08X",image.getRGB(image.getWidth() - 1, 0)));
+                    System.out.println("Robot.createScreenCapture Bottom Right Actual: " +
+                            String.format("0x%08X",image.getRGB(image.getWidth() - 1, image.getHeight() - 1)));
+                    System.out.println("Robot.createScreenCapture Top Left Actual: " +
+                            String.format("0x%08X",image.getRGB(0, image.getHeight() - 1)));
```

Comment:
There's no space after the comma, after the format string. Yet there are spaces in `getRGB`, probably because they're copied from the `if`-condition.